aptos-move/framework/aptos-framework/sources/block.spec.move

```diff
@@ -118,6 +127,12 @@ spec aptos_framework::block {
         aborts_if !exists<BlockResource>(addr);
         let post block_resource = global<BlockResource>(addr);
         ensures block_resource.epoch_interval == new_epoch_interval;
+
+        // Only the Aptos framework address can execute
+        ensures addr == @aptos_framework;
+
+        // epoch_interval > 0
+        ensures block_resource.epoch_interval > 0;
```

Comment:
I don' think it is necessary because a data invariant `invariant epoch_interval > 0;` is already defined above.

Reply:
have removed